myplanet/skeletor
# SKEL-17: Media object component

JIRA Story: https://myplanet.jira.com/browse/SKEL-17

### Testing Notes
* Create a clean site build
* Make the "Bare Bones Bootstrap Starterkit" the default theme
* At the root of the theme, run ``yarn install`` && ``yarn build``
* Enable the "Skeletor Media Object Component" (``drupal moi skeletor_media_object --yes``) module
* Create a content type
* Add a paragraph entity revision field, and select the "Media List Component"
* Create a new node
* Add a few media objects and save node. Results should look like [Bootstrap media object styling](https://getbootstrap.com/docs/4.1/layout/media-object).
* Add **another** paragraph entity revision field, and select the "Media List Component"
* On the manage form display tab, change the form mode to "Media Nested"
* On the manage display tab, change the view mode to "Media nested"
* Edit the node your create, and add a few media objects, along with **at least one nested media object**. Results should look like [Bootstrap nested media object styling](https://getbootstrap.com/docs/4.1/layout/media-object#nesting).
